### PR TITLE
fixed bulid error

### PR DESCRIPTION
**Pull request by kostaDimitrijevic**

### CODE REVIEW

File: source/dagger/gameplay/ping_pong/ping_pong_main.cpp [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #include "gameplay/ping_pong/pingpong_ball.h"
 #include "gameplay/ping_pong/player_scores.h"
 #include "gameplay/ping_pong/pingpong_playerinput.h"
-#include "gameplay/ping_pong/impact_flash.h"
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/ping_pong_main.cpp:21]: can you please get it back

File: source/dagger/gameplay/ping_pong/ping_pong_main.cpp [hunk 2]
```diff
@@ -66,20 +66,27 @@ void PingPongGame::GameplaySystemsSetup(Engine& engine_)
     engine_.AddSystem<PingPongBallSystem>();
     engine_.AddSystem<PingPongPlayerInputSystem>();
     engine_.AddSystem<PlayerScoresSystem>();
-    engine_.AddSystem<ImpactFlashSystem>();
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/ping_pong_main.cpp:69]: return this pls

File: source/dagger/gameplay/ping_pong/ping_pong_main.cpp [hunk 3]
```diff
@@ -228,8 +235,6 @@ void PingPongGame::WorldSetup(Engine& engine_)
 
         auto& controller = reg.emplace<ControllerMapping>(entity);
         PingPongPlayerInputSystem::SetupPlayerOneInput(controller);
-
-        reg.emplace<ImpactFlash>(entity);
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/ping_pong_main.cpp:232]: this

File: source/dagger/gameplay/ping_pong/ping_pong_main.cpp [hunk 4]
```diff
@@ -251,10 +256,8 @@ void PingPongGame::WorldSetup(Engine& engine_)
 
         auto& controller = reg.emplace<ControllerMapping>(entity);
         PingPongPlayerInputSystem::SetupPlayerTwoInput(controller);
-
-        reg.emplace<ImpactFlash>(entity);
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/ping_pong_main.cpp:255]: this

**Review by DPrivalov — CHANGES_REQUESTED** [review]

**Review by smiljanaspasic — APPROVED** [review]

**Pull request merged**